src/index.html

```diff
@@ -3,12 +3,356 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dia</title>
+    <title>Air</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:400,600,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Dia</h1>
+    <header class="header" id="_home">
+      <div class="header__wrapper">
+        <nav class="nav">
+          <ul class="nav__list">
+            <li>
+              <a href="#_home" class="nav__link">
+                <img src="images/logo/Logo.png" alt="air logo">
+              </a>
+            </li>
+            <li>
+              <a href="#_home" class="nav__link">Home</a>
```

Comment:
подчеркивание не нужно
```suggestion
              <a href="#home" class="nav__link">Home</a>
```